__tests__/tools/sanitze-filename.js

```diff
@@ -0,0 +1,13 @@
+const sanitizeFilename = require('../../src/utils/sanitize-filename');
+
+describe('sanitizeFilename method', () => {
+  it('should remove emojis from filename', () => {
+    expect(sanitizeFilename('🇯🇵japan.😢jpg')).toBe('japan.jpg');
```

Comment:
Can we use the emoji in the extension?

```suggestion
    expect(sanitizeFilename('🇯🇵japan😢.jpg')).toBe('japan.jpg');
```

Reply:
What do you mean?

Reply:
the emoji is set after the dot, was that intentional?

Reply:
Yes, I want to test that the emoji is removed from the name and the extension as well.